dotCMS/src/main/java/com/dotmarketing/portlets/director/action/DirectorAction.java

```diff
@@ -688,15 +688,15 @@ else if (workingLink.isLocked()) {
 	
 						if( newPosition == x ) {
 							iden = APILocator.getIdentifierAPI().find(contentlet);
-							multiTree = MultiTreeFactory.getMultiTree(idenHtmlPage,idenContainer,iden);
+							multiTree = MultiTreeFactory.getMultiTree(idenHtmlPage,idenContainer,iden, MultiTree.LEGACY_RELATION_TYPE);
 							multiTree.setTreeOrder(x);
 							MultiTreeFactory.saveMultiTree(multiTree, htmlPage.getLanguageId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "saveMultiTree"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)